projects/plugins/automattic-for-agencies-client/src/js/components/branded-card/styles.module.scss

```diff
@@ -1,3 +1,9 @@
+// to do: replace after @automattic/color-studio@2.6.0 release
+:root {
+	--automattic-blue-60: #0088BF;
+	--automattic-blue-80: #021A23;
+}
```

Comment:
Temporary solution until the full color scheme is available via package.

Reply:
It should be already, since #36751.

Reply:
Nice! Added color studio in https://github.com/Automattic/jetpack/pull/36747/commits/d37902ccd4baf1a60d1535eb7438642ac4c333e0.